rnd/autogpt_builder/src/hooks/useAgentGraph.ts

```diff
@@ -608,7 +614,7 @@ export default function useAgentGraph(
           }));
 
         return {
-          id: node.id,
+          id: node.data.backend_id,
```

Comment:
This is breaking the execution of blocks. I believe this should be switched to existing value

Reply:
Thank you for flagging this!
I have created a follow-up PR to fix it, let me know if the issue persists:
https://github.com/Significant-Gravitas/AutoGPT/pull/8054